### ISSUE TITLE
BuildBot - Insecure connection error.

### ISSUE THREAD

**Issue by JhonLoveJoy** [hrydgard/ppsspp#10405]
![image](https://user-images.githubusercontent.com/31490343/33941217-ea14153e-dff8-11e7-9e87-2ec359f43b03.png)

What's happening?

**Comment by hrydgard** [comment]
@JhonLoveJoy , strange, for me the site loads up find and I can download builds without problem. @Orphis , any idea?

@marosis Are you saying that the latest build from the buildbot crash if you play them on your android device? That would be a different issue than this, which seems to be about the website (one you should report!)

**Comment by unknownbrackets** [comment]
Not sure what OS this is on, but some devices don't support the SSL:
https://letsencrypt.org/docs/certificate-compatibility/

Also, SNI or IPv6 is probably required... but if you're on Vista or newer, or on Android 4 or newer, or any other OS that's not ancient, it should work.

-[Unknown]

**Comment by Orphis** [comment]
Looks like a MITM to me, his browser seems to be recent enough.

It would be interesting to see the certificate information you see by clicking on the padlock and copying all the information there. In layman's terms, you may have someone intercepting your browsing to the website and changing content.

**Comment by marosis** [comment]
I tried Now And not crashing. I don't know why

**Comment by JhonLoveJoy** [comment]
I don't know what happened this morning. But now it's working normally again.

**Comment by marosis** [comment]
Orphis please can you look at this? #10118

**Comment by Orphis** [comment]
@JhonLoveJoy Were you at home on your own wifi network or in a public one?

**Comment by JhonLoveJoy** [comment]
@Orphis - I was using the network from my house.

**Comment by Orphis** [comment]
Is it still happening?
You should try to reinstall Firefox or try Chrome to see if they report the same issue.
If you have a laptop, it would be great to try it from some other wifi access point to.

**Comment by JhonLoveJoy** [comment]
Nop. It just happened that time. 

Now everything is working normally, I can not explain what happened. .-.

**Comment by unknownbrackets** [comment]
Okay, I'm going to close now (you're still able to comment), as it seems to be resolved.

This could just mean a DNS problem.  For example, sometimes routers (when the internet is unavailable) reroute all IP addresses to the router, in order to display an "internet unavailable" message.  It may have been that.

-[Unknown]
